doc/source/api_reference/graphicsItems/imageitem.rst

```diff
@@ -53,13 +96,16 @@ Examples
 .. literalinclude:: /images/gen_example_imageitem_transform.py
```

Comment:
I was thinking this would be a good place to insert examples that demonstrate "simple and common" operations that may not be obvious,   I've viewed as examples as being typically more complicated... and probably shouldn't go right there... that said the examples app certainly does need more work!

Reply:
I moved that first paragraph of the notes section down, but given that that paragraph talks about `setRect` which the example doesn't demonstrate, figured I'd just keep it in the notes section.  I added a blurb before the example to better explain it.  Hopefully that's sufficient.

Reply:
> I was thinking this would be a good place to insert examples that demonstrate "simple and common" operations that may not be obvious, I've viewed as examples as being typically more complicated... and probably shouldn't go right there... that said the examples app certainly does need more work!

Definitely. The short-and-simple examples could go in a different place to not clutter the "demos" gallery/app. An alternative might be a tutorial style example where you can make various manipulations and show the effects of various options iteratively. e.g. https://github.com/matplotlib/matplotlib/blob/main/galleries/tutorials/images.py -> https://matplotlib.org/stable/tutorials/images.html#sphx-glr-tutorials-images-py

Reply:
Those kinds of more "interactive" tutorials would likely better served by something that is rendered in jupyter-sphinx (which should be feasible since we work with the jupyter_rfb extension)